site/ruby_scripts/tool.rb

```diff
@@ -0,0 +1,107 @@
+# encoding: utf-8
```

Comment:
Probably good to add a comment at top of the file that this file is adapted from here: https://github.com/feedreader/jekyll-planet/blob/master/lib/jekyll/planet/tool.rb

This should be useful for maintainability

Also adding that the original is released under the public domain: https://github.com/feedreader/jekyll-planet#license